core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -24,110 +24,144 @@
+      put("alluxio.user.network.netty.channel.pool.size.max", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.channel.pool.size.min", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.reader.buffer.size.packets", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.reader.packet.size.bytes", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.timeout", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.writer.buffer.size.packets", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.writer.close.timeout", removedSince(V2_0_0));
+      put("alluxio.user.network.netty.writer.packet.size.bytes", removedSince(V2_0_0));
+      put("alluxio.user.network.socket.timeout", removedSince(V2_0_0));
+      put("alluxio.user.rpc.retry.max.num.retry", removedSince(V2_0_0));
+      put("alluxio.user.ufs.delegation.read.buffer.size.bytes", removedSince(V2_0_0));
+      put("alluxio.user.ufs.delegation.write.buffer.size.bytes", removedSince(V2_0_0));
+      put("alluxio.user.ufs.file.reader.class", removedSince(V2_0_0));
+      put("alluxio.user.ufs.file.writer.class", removedSince(V2_0_0));
+      put("alluxio.web.temp.path", removedSince(V2_0_0));
+      put("alluxio.worker.block.threads.max", removedSince(V2_0_0));
+      put("alluxio.worker.block.threads.min", removedSince(V2_0_0));
+      put("alluxio.worker.data.bind.host", removedSince(V2_0_0));
+      put("alluxio.worker.data.hostname", removedSince(V2_0_0));
+      put("alluxio.worker.data.port", replacedSince(V2_0_0, PropertyKey.WORKER_RPC_PORT.getName()));
+      put("alluxio.worker.network.netty.async.cache.manager.threads.max", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.backlog", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.block.reader.threads.max", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.block.writer.threads.max", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.buffer.receive", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.buffer.send", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.file.transfer", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.file.writer.threads.max", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.reader.buffer.size.packets", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.rpc.threads.max", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.shutdown.timeout", removedSince(V2_0_0));
+      put("alluxio.worker.network.netty.writer.buffer.size.packets", removedSince(V2_0_0));
+      put("alluxio.worker.tieredstore.reserver.enabled", removedSince(V2_0_0));
+      put("alluxio.worker.tieredstore.retry", removedSince(V2_0_0));
+      put("fs.swift.apikey", replacedSince(V2_0_0, PropertyKey.Name.SWIFT_PASSWORD_KEY));
+      put("fs.swift.use.public.url", removedSince(V2_0_0));
```

Comment:
@madanadit  I found this property `fs.swift.use.public.url` is referenced quite a bit in doc but never used in code (even in 1.8), can you double check if it is safe to delete this property?

Reply:
confirmed, we are not using this property key any more